xc7/archs/zynq7/devices/xc7z010-roi-virt/dummy/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+function(GENERATE_DUMMY)
+  set(DUMMY_FILES
+    CLBLL_R.xml CLBLM_L.xml CLBLM_R.xml
+    INT_L.xml INT_R.xml HCLK_L.xml HCLK_R.xml)
+
```

Comment:
Why are you generating dummies for these?

Reply:
Actually I had a question about the purpose about this dummy folder

Reply:
It's no longer used, safe to delete